README.md

```diff
@@ -1,19 +1,105 @@
 # toffy
-Scripts for interacting with and generating data from the commercial MIBIScope
+Scripts for interacting with and generating data from the commercial MIBIScope.
 
+# System
+Depending on what operating system you are planning to install toffy on, the process will differ slightly.
+- [Install on CAC](#cac)
+- [Install on Windows](#windows)
```

Comment:
The CAC is a windows machine, why do we have separate sections for what is essentially the same thing?

Reply:
The only difference is the instructions for the admin password so I'll just not that under the windows section